### PR TITLE
[py3] Add jinja2-filter tests

### PR DESCRIPTION
**Pull request by ZachOrr**
Just adding some missing tests. Note - the `register_template_filters` function has changed a little bit.

### CODE REVIEW

File: stubs/_pytest/mark/__init__.pyi [hunk 1]
```diff
@@ -4,3 +4,4 @@ from .structures import (
     MarkGenerator as MarkGenerator,
     get_empty_parameterset_mark as get_empty_parameterset_mark,
 )
+from .structures import MARK_GEN
```
Comment (ZachOrr) [inline stubs/_pytest/mark/__init__.pyi:7]: cc @phil-lopreiato getting this error when running pyre
Screen Shot 2020-05-30 at 1 06 45 PM

Weirdly when I do a stubgen I get wildly different generated files then what's in the repo. Are we using different versions of things?

```
$ pytest --version
This is pytest version 5.4.2, imported from /Users/zach/Desktop/the-blue-alliance/venv/lib/python3.7/site-packages/pytest/__init__.py
setuptools registered plugins:
  pytest-cov-2.9.0 at /Users/zach/Desktop/the-blue-alliance/venv/lib/python3.7/site-packages/pytest_cov/plugin.py
$ mypy --version
mypy 0.770
```

File: src/web/tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from flask import Flask
+import pytest
+
+
+@pytest.fixture
+def app() -> Flask:
```
Comment (phil-lopreiato) [inline src/web/tests/conftest.py:6]: nit: name this something other than just `app`? Maybe `empty_app`?

File: stubs/_pytest/mark/__init__.pyi (hunk 1 above)
Comment (phil-lopreiato) [reply stubs/_pytest/mark/__init__.pyi:7]: seems like this is resolved in the latest push?

Comment (ZachOrr) [reply stubs/_pytest/mark/__init__.pyi]: It's not :\

**Pull request merged**